src/qwed_open_responses/guards/safety_guard.py

```
@@ -183,8 +183,53 @@ def check(
+        - output/arguments dicts were already stringified above — skipping
+          avoids duplicate collection.
+        """
+        if isinstance(value, str):
+            # content/output/text strings were collected verbatim above;
+            # a string under 'arguments' was NOT (only dicts are) and must
+            # still be scanned for injection/PII.
+            if key == "arguments":
+                return True
+            return key not in self._KNOWN_CONTENT_KEYS
+        if isinstance(value, dict):
+            return not (key in ("output", "arguments"))
```

Ruff (0.16.2)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)